src/vm/arm64/asmhelpers.S

```diff
@@ -121,18 +121,18 @@ LEAF_END HelperMethodFrameRestoreState, _TEXT
 // The call in ndirect import precode points to this function.
 NESTED_ENTRY NDirectImportThunk, _TEXT, NoHandler
 
-    PROLOG_SAVE_REG_PAIR_INDEXED fp, lr, -160
+    PROLOG_SAVE_REG_PAIR_INDEXED fp, lr, -240
```

Comment:
@CarolEidt I am trying to figure out why the frame size has changed to 240. Argument registers occupy sp+16.. sp+87, then you have added 8 bytes of padding to align the float registers address to 16 bytes and then you have 128 bytes of the floating point arg registers. 96+128=224, but you have 240 there.

Reply:
That's a good question, and I can't recall the reason. It may be an arithmetic error, but I made those changes in my first round of working on that, and it's been awhile so I don't recall (this is why I try to do a better job of writing comments!). Let me look into this.

Reply:
I think this was just a mistake. I've changed it to 224.